client/components/clipboard-button-input/index.jsx

```diff
@@ -12,35 +14,34 @@ import { withoutHttp } from 'lib/url';
 import ClipboardButton from 'components/forms/clipboard-button';
 import FormTextInput from 'components/forms/form-text-input';
 
-export default React.createClass( {
-	displayName: 'ClipboardButtonInput',
-
-	propTypes: {
-		value: PropTypes.string,
-		disabled: PropTypes.bool,
-		className: PropTypes.string,
-		hideHttp: PropTypes.bool
-	},
-
-	getInitialState() {
-		return {
+class ClipboardButtonInputExport extends React.Component {
+	constructor( props ) {
```

Comment:
feel free to ignore:

If the only thing a `constructor` is doing is setting initial state then you can do it without the constructor in the format:

```js
class ClipboardButtonInputExport extends React.Component {
  state = {
    ...
  }
}
```

source: https://babeljs.io/blog/2015/06/07/react-on-es6-plus

Reply:
Nice one, thanks for sharing :) It's great that they are trying to get rid of any usages of constructor!